scalafmt-dynamic/src/main/scala/org/scalafmt/dynamic/ScalafmtVersion.scala

```diff
@@ -34,17 +36,18 @@ case class ScalafmtVersion(
 object ScalafmtVersion {
 
   private val versionRegex =
-    """(\d{1,2})\.(\d{1,2})\.(\d{1,2})(?:-RC(\d{1,2}))?([+].*|-SNAPSHOT)?""".r
```

Comment:
wouldn't this original regex parse the hashes as well, but simply put them in snapshot? i.e., using test below:
```
   assertEquals(
      ScalafmtVersion.parse("3.5.9+41-60b84efb-SNAPSHOT"),
      Some(ScalafmtVersion(3, 5, 9, 0, "+41-60b84efb-SNAPSHOT"))
    )
```
and if so, what does this change enable if parsing worked already?

Reply:
Ahhh, you're right, it should work without my change...

Reply:
Maybe @mlachkar uses older version of scalafmt? (older than https://github.com/scalameta/scalafmt/commit/f7568e9410efa4dff9280ef3fcd27f796af524dd)